python/cudf/cudf/_fuzz_testing/tests/fuzz_test_csv.py

```diff
@@ -6,33 +6,121 @@
 import pandas as pd
 
 import cudf
-from cudf.testing.csv import CSVReader, CSVWriter
-from cudf.testing.main import pythonfuzz
-from cudf.testing.utils import compare_content, run_test
+from cudf._fuzz_testing.csv import CSVReader, CSVWriter
+from cudf._fuzz_testing.main import pythonfuzz
+from cudf._fuzz_testing.utils import compare_content, run_test
 from cudf.tests.utils import assert_eq
 
 
 @pythonfuzz(data_handle=CSVReader)
 def csv_reader_test(csv_buffer):
-    pdf = pd.read_csv(csv_buffer)
-    gdf = cudf.read_csv(csv_buffer)
+    pdf = pd.read_csv(StringIO(csv_buffer))
+    gdf = cudf.read_csv(StringIO(csv_buffer))
 
     assert_eq(gdf, pdf)
 
 
 @pythonfuzz(data_handle=CSVWriter)
-def csv_writer_test(gdf):
-    pdf = gdf.to_pandas()
+def csv_writer_test(pdf):
+    gdf = cudf.from_pandas(pdf)
 
     pd_buffer = pdf.to_csv()
     gd_buffer = gdf.to_csv()
 
     compare_content(pd_buffer, gd_buffer)
-
     actual = cudf.read_csv(StringIO(gd_buffer))
     expected = pd.read_csv(StringIO(pd_buffer))
     assert_eq(actual, expected)
 
 
+@pythonfuzz(
+    data_handle=CSVWriter,
+    params={
+        "sep": list([","]),
+        "header": [True, False],
+        "na_rep": [
+            "",
+            "<NA>",
+            "NA",
+            "_NA_",
+            "__",
+            "<<<<>>>>>",
+            "--<>--",
+            "-+><+-",
+        ],
+        "columns": None,
+        "index": [True, False],
+        "line_terminator": ["\n", "\r"],
+        "chunksize": None,
+    },
+)
+def csv_writer_test_params(
+    pdf, sep, header, na_rep, columns, index, line_terminator, chunksize
+):
+    gdf = cudf.from_pandas(pdf)
+
+    pd_buffer = pdf.to_csv(
+        sep=sep,
+        header=header,
+        na_rep=na_rep,
+        columns=columns,
+        index=index,
+        line_terminator=line_terminator,
+        chunksize=chunksize,
+    )
+    gd_buffer = gdf.to_csv(
+        sep=sep,
+        header=header,
+        na_rep=na_rep,
+        columns=columns,
+        index=index,
+        line_terminator=line_terminator,
+        chunksize=chunksize,
+    )
+
+    # compare_content(pd_buffer, gd_buffer)
```

Comment:
Is this stale code or is this needed?

Reply:
Needed, but disable for now due to non-identical CSV file contents because of:  https://github.com/rapidsai/cudf/issues/6418
Added a comment to make it clear.